training/nodejs/js/index.js

```diff
@@ -1 +1,29 @@
-console.log("Hello World!")
\ No newline at end of file
+const https = require('https') 
+
+const FEED_URl = "https://api.nasa.gov/neo/rest/v1/feed"
+const START_DATE = "2024-02-26"
+const END_DATE = "2024-03-01"
+const API_KEY = "KhS7EQVPfk4NLgak8tr7LgNjVNbI0Feuj4q3cMC3"
+
+const buildUrl = (url, startDate, endDate, apiKey) => {
+    return `${url}?start_date=${startDate}&end_date=${endDate}&api_key=${apiKey}`
+}
+
+const getAsteroidsWithinPeriod = (startDate, endDate) => {
```

Comment:
I am not an expert in JS, but as I have read we can also use libraries for making HTTP calls simpler. For example,  Axios 
https://blog.logrocket.com/5-ways-make-http-requests-node-js/